src/App.js

```diff
@@ -1,17 +1,18 @@
+/* eslint-disable */
 import React from 'react';
 import { Routes, Route } from 'react-router-dom';
 import './App.css';
-import Nav from './components/Nav';
-import Book from './components/Book';
-import Categories from './components/categories';
+import Navbar from './components/Navbar';
+import Books from './components/Books';
+import Categories from './components/Categories';
```

Comment:
- [ ] Please 🙏🏽 correct the path to the categories to avoid issues. It is `'./components/categories'` all in lowercase instead of `'./components/Categories'`

![image](https://github.com/Javed-Shahbaz/book-store-react/assets/58642949/9fd24a07-25cf-4de6-97a4-e961b343cb75)